spec/support/features/doc_auth_helper.rb

```diff
@@ -81,7 +81,7 @@ def complete_how_to_verify_step(remote: true)
       app_name: APP_NAME,
     )
     find(
-      'label',
+      'label.usa-radio__label',
       text: text_for_method,
```

Comment:
Added class selector because I was getting error `Capybara::Ambiguous: Ambiguous match, found 2 elements matching visible css "label" with text "Verify your identity at a Post Office"`